src/ui/header/mobile-navigation.tsx

```
@@ -0,0 +1,190 @@
+    <div 
+      className="fixed inset-0 top-[57px] z-50 overflow-hidden bg-background/95 border-foreground/10"
+      role="dialog"
+      aria-modal="true"
+      aria-label="Mobile navigation menu"
+    >
+      <nav className="h-full overflow-y-auto" aria-label="Mobile navigation">
+        <div className="mx-auto max-w-screen-xl p-4 space-y-6">
+          <div className="flex items-center justify-between">
+            <CTAList ctas={ctas} className="grid flex-1 gap-2 *:w-full" />
+          </div>
+          <div className="h-px bg-border" role="separator" />
```

Biome (1.9.4)

[error] The HTML element with the interactive role "separator" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "separator".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)